recipes/google-compute-engine/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "google-compute-engine" %}
+{% set version = "2.8.13" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 358363a10169f890bac78cf9d7105132cdd2c1546fa8d9caa35c04a7cf7cfba7
+
+build:
+  number: 0
+  entry_points:
+    - google_accounts_daemon=google_compute_engine.accounts.accounts_daemon:main
+    - google_clock_skew_daemon=google_compute_engine.clock_skew.clock_skew_daemon:main
+    - google_instance_setup=google_compute_engine.instance_setup.instance_setup:main
+    - google_network_daemon=google_compute_engine.networking.network_daemon:main
+    - google_metadata_script_runner=google_compute_engine.metadata_scripts.script_manager:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win]
+
+requirements:
+  host:
+    - boto
+    - distro
+    - pip
+    - python
+    - setuptools
+  run:
+    - boto
+    - distro
+    - python
+    - setuptools
+
+test:
+  imports:
+    - google_compute_engine
+    - google_compute_engine.accounts
+    - google_compute_engine.boto
+    - google_compute_engine.clock_skew
+    - google_compute_engine.distro_lib
+    - google_compute_engine.distro_lib.debian_8
+    - google_compute_engine.distro_lib.debian_8.tests
+    - google_compute_engine.distro_lib.debian_9
+    - google_compute_engine.distro_lib.debian_9.tests
+    - google_compute_engine.distro_lib.el_6
+    - google_compute_engine.distro_lib.el_6.tests
+    - google_compute_engine.distro_lib.el_7
+    - google_compute_engine.distro_lib.el_7.tests
+    - google_compute_engine.distro_lib.freebsd_11
+    - google_compute_engine.distro_lib.freebsd_11.tests
+    - google_compute_engine.distro_lib.sles_11
+    - google_compute_engine.distro_lib.sles_11.tests
+    - google_compute_engine.distro_lib.sles_12
+    - google_compute_engine.distro_lib.sles_12.tests
+    - google_compute_engine.instance_setup
+    - google_compute_engine.metadata_scripts
+    - google_compute_engine.networking
+    - google_compute_engine.networking.ip_forwarding
+    - google_compute_engine.networking.ip_forwarding.tests
+    - google_compute_engine.networking.network_setup
+    - google_compute_engine.networking.network_setup.tests
+    - google_compute_engine.networking.tests
+  commands:
+    - google_accounts_daemon --help
+    - google_clock_skew_daemon --help
+    - google_instance_setup --help
+    - google_network_daemon --help
+    - google_metadata_script_runner --help
+
+about:
+  home: https://github.com/GoogleCloudPlatform/compute-image-packages
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Google Compute Engine
+  doc_url: https://cloud.google.com/compute/docs/
+  dev_url: https://pypi.io/packages/source/g/google-compute-engine/google-compute-engine-2.8.13.tar.gz
```

This should point to the repository/webpage used for development:
```suggestion
  dev_url: https://github.com/GoogleCloudPlatform/compute-image-packages/tree/master/packages/python-google-compute-engine
```